modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -727,10 +724,6 @@ GDScriptParser::DataType GDScriptAnalyzer::resolve_datatype(GDScriptParser::Type
 		}
 	}
 
-	if (result.builtin_type != Variant::ARRAY && p_type->container_type != nullptr) {
-		push_error("Only arrays can specify the collection element type.", p_type);
```

Comment:
Is there other types than arrays that can have a container type? What led to the removal of this part?

Reply:
Misunderstanding on my part. Put it back now.